invenio_oauthclient/models.py

```diff
@@ -146,10 +146,10 @@ class RemoteToken(db.Model):
     secret = db.Column(db.Text(), default='', nullable=False)
     """Used only by OAuth 1."""
 
-    def __repr__(self):
+    def __repr__(self):  # pragma no cover
```

Comment:
Sorry, assumed it was good practice as I saw it as an example on the documentation of coverage.py (http://coverage.readthedocs.io/en/coverage-4.0.3/excluding.html). Anyway, updated.